analyzer.go

```diff
@@ -12,18 +14,19 @@ import (
 	types "github.com/gogo/protobuf/types"
 	"github.com/src-d/lookout"
 	log "gopkg.in/src-d/go-log.v1"
+	"gopkg.in/src-d/lookout-sdk.v0/pb"
```

Comment:
I think we should also add `lookout-sdk` into [`Gopkg.toml`](https://github.com/src-d/lookout-gometalint-analyzer/blob/master/Gopkg.toml), shouldn't we?

```toml
[[constraint]]
  name = "gopkg.in/src-d/lookout-sdk.v0"
```

Reply:
it's part of TODO. Replace lookout with lookout-sdk in dep. But we need new version of sdk first :)